CONTRIBUTING.md

```diff
@@ -311,6 +309,15 @@ for the previous version. For example, if the latest, published release is
 v1.2, then minor changes for the previous release—e.g., v1.1.0 -> v1.1.1—are
 still possible.
 
+The previous release gets fixes for:
+* Security issues
```

Comment:
Could you use a `-` for the lists instead of `*`? This keeps all markdown lists consistent.

Reply:
done

Reply:
Thanks :)

Reply:
That's necessary, otherwise it won't pass markdown lint .....

ps: there are too much content in this file, I think it's time to  split it.